ampd/src/queue/queued_broadcaster.rs

```diff
@@ -253,6 +253,108 @@ mod test {
         assert!(handle.await.unwrap().is_ok());
     }
 
+    #[test(start_paused = true)]
+    async fn should_broadcast_after_interval_in_low_load() {
```

Comment:
the test is doing the right thing now, but can be improved to get rid of the sleep. see if you can figure out how to use a combination of a oneshot channel https://docs.rs/tokio/latest/tokio/sync/oneshot/ and a timeout https://docs.rs/tokio/latest/tokio/time/fn.timeout.html to remove it